account_statement_import_camt54/models/parser.py

```diff
@@ -54,35 +17,26 @@ def parse_transaction_details(self, ns, node, transaction):
         # put the esr in the label. odoo reconciles based on the label,
         # if there is no esr it tries to use the information textfield
 
-        isr_number = node.xpath(
-            "./ns:RmtInf/ns:Strd/ns:CdtrRefInf/ns:Ref", namespaces={"ns": ns}
```

Comment:
in the file camt054,  "**./ns:RmtInf/ns:Strd/ns:CdtrRefInf/ns:Ref**" contains the QRR code, NOT isr_number. So, don't remove this part but rename it to `qrr_number`. not only here, but every where inside this module, replace `isr` by `qrr`
![image](https://github.com/sonhd91/bank-statement-import/assets/8724793/92906bba-cb91-43cc-bb2e-0e5178ee1f29)